### ISSUE TITLE
SVG Shapes not working in IE http://fabricjs.com/kitchensink

### ISSUE THREAD

**Issue by shaktippatel** [fabricjs/fabric.js#5218]
SVG Shapes not working in IE

## Version
2.3.5

## Test Case
http://fabricjs.com/kitchensink

**Comment by shaktippatel** [comment]
I have checked found that ordrer changed in makeXHR function
```
      function() { return new fabric.window.XMLHttpRequest(); },
      function() { return new ActiveXObject('Microsoft.XMLHTTP'); },
      function() { return new ActiveXObject('Msxml2.XMLHTTP'); },
      function() { return new ActiveXObject('Msxml2.XMLHTTP.3.0'); }
```

**Comment by asturur** [comment]
closed via #5307
